OSVR-Unity/Assets/OSVRUnity/src/ClientKit.cs

```diff
@@ -30,6 +30,7 @@ public class ClientKit : MonoBehaviour
             public string AppID;
 
             private OSVR.ClientKit.ClientContext contextObject;
+            private static ClientKit _instance;
```

Comment:
Do we need to have an ` = null` in there to ensure null initialization, or is that automatic with C#?